docs/machine-learning/tutorials/image-classification-model-builder.md

```diff
@@ -0,0 +1,247 @@
+1. In the Create New Compute dialog, type "landuse-cpt" in the **Compute name** text box.
+1. Choose **Standard_NC24** from the **Compute size** dropdown. Model Builder uses GPU-optimized compute types. Visit the [NC-series Linux VM documentation](https://docs.microsoft.com/azure/virtual-machines/nc-series?toc=/azure/virtual-machines/linux/toc.json&bc=/azure/virtual-machines/linux/breadcrumb/toc.json) for more details on GPU optimized compute types.
+1. Select **Create**. The compute resources may take a few minutes to provision.
+1. Once the provisioning process is complete, choose your newly created workspace from the **Compute name** dropdown in the Create New Experiment dialog.
+1. Select the **Next step** button to load in the data.
+
+## Load the data
+
+1. In the data step of the Model Builder tool, select the button next to the **Select a folder** text box.
+1. Use File Explorer to browse and select the unzipped directory containing the images.
+1. Select the **Next step** button to move to the next step in the Model Builder tool.
+
+## Train the model
+
+Training on Azure is only available for the Model Builder image classification scenario. The algorithm used to train these models is a Deep Neural Network based on the ResNet50 architecture. During the model training process, Model Builder trains separate models using ResNet50 algorithm and settings to find the best performing model for your dataset.
+
+### Start training
+
+Once you've configured your workspace and compute type, it's time to finish creating the experiment and start training.
+
+1. Click the **Start Training** button.
+
+    The training process takes some time and the amount of time may vary depending on the size of compute selected as well as amount of data. The first time a model is trained, you can expect a slightly longer training time because resources have to be provisioned. You can track the progress of your runs by selecting the **Monitor current run in Azure portal** link in Visual Studio.
+
+    Throughout the training process, progress data is displayed in the Progress section of the train step.
+
+    - Status displays the completion status of the training process.
+    - Best accuracy displays the accuracy of the best performing model found by Model Builder so far. Higher accuracy means the model predicted more correctly on test data.
+    - Algorithm displays the name of the best performing algorithm performed found by Model Builder so far.
+
+1. Once training is complete, select the **Next step** button to move to evaluate the model.
+
+### Evaluate the model
+
+The result of the training step is one model that had the best performance. In the evaluate step of the Model Builder tool, the **Details** tab in the output section, will contain the algorithm used by the best performing model in the **Algorithm** entry along with metrics in the **Best model Accuracy** entry.
```

Comment:
Is the capitalization of "Best model Accuracy" correct?

Reply:
It's shown like this
![image](https://user-images.githubusercontent.com/1578160/134892993-a128182b-4db3-4a46-ab0d-7c7be2610c34.png)

I'm guessing we can just say "Best model accuracy" though or do you think there is a better way to word it?

Reply:
Accuracy in the Best Model details